frontend/src/views/ManureAndImports/ManureAndImports.tsx

```diff
@@ -231,16 +217,15 @@ export default function ManureAndImports() {
         width: 200,
         minWidth: 150,
         maxWidth: 300,
-        valueGetter: (param: string | number) => (param === '1' ? 'Beef Cattle' : 'Dairy Cattle'),
+        valueGetter: (val: any) => animals.find((ele) => String(ele.id) === val)?.name || val,
       },
       {
-        field: 'subtype',
+        field: 'manureId',
         headerName: 'Animal Sub Type',
         width: 325,
         minWidth: 150,
         maxWidth: 500,
-        valueGetter: (param: string | number) =>
-          cattleSubtypeList?.find((ele) => ele.id === param)?.label || param,
+        valueGetter: (val: string) => animalList.find((a) => a.manureId === val)!.manureData?.name,
```

Comment:
This is a dumb way to get the subtype, but I don't want to worry about the other animals subtype id. The name already gets saved as the manure data name. Maybe it should get saved to a different attribute.